src/v2/Apps/Auction/Components/Form/helpers.tsx

```diff
@@ -14,8 +28,25 @@ const ERROR_SUFFIX = {
   "Your card's security code is incorrect.": "Please try again.",
 }
 
-export const errorMessageForCard = (errorMessage: string) => {
-  return `${errorMessage} ${ERROR_SUFFIX[errorMessage] || ""}`
+const BIDDING_STATE_TO_MESSAGE_MAPPING = {
```

Comment:
I moved this constant to the modal component but moved it back to this file. It's awkward if the helper file imports a constant from a view component because components are practically the ones that depend on helpers, but not the other way around. However, there is definitely room for improvement here.